compiler/rustc_codegen_llvm/src/debuginfo/create_scope_map.rs

```diff
@@ -102,8 +102,8 @@ fn make_mir_scope(
                 DIB(cx),
                 parent_scope.dbg_scope.unwrap(),
                 file_metadata,
-                loc.line.unwrap_or(UNKNOWN_LINE_NUMBER),
-                loc.col.unwrap_or(UNKNOWN_COLUMN_NUMBER),
+                loc.line.map(|n| n.get()).unwrap_or(UNKNOWN_LINE_NUMBER),
```

Comment:
`map_or` would be a better fit here. Same for all the other similar lines.

Reply:
Completely forgot about `map_or`, thanks!